recipes/gunicorn/no_gaiohttp.patch

```diff
@@ -0,0 +1,174 @@
+diff --git a/gunicorn/workers/_gaiohttp.py b/gunicorn/workers/_gaiohttp.py
+deleted file mode 100644
+index cdce4be..0000000
+--- a/gunicorn/workers/_gaiohttp.py
```

Comment:
That is "correct" it is just the level that it is applying that is not OK for old conda-build. You won't have to worry about that in the future.